src/core/sqs-managed-sse-status-advisor.ts

```diff
@@ -0,0 +1,14 @@
+import { Advisor } from "./types";
+import { Worker } from "./worker";
+
+export class SqsManagedSseStatusAdvisor implements Advisor {
+  apply(worker: Worker) {
+    const suggestions: string[] = [];
+    if (worker.sqs.Properties.SqsManagedSseEnabled === false) {
```

Comment:
Recheck this [KMS-SQS](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sqs-queue.html#cfn-sqs-queue-kmsmasterkeyid)